extra/1-factorial.js

```diff
@@ -9,7 +9,18 @@
 */
 
 function factorial(input) {
-  // TODO
+  if (input == 0) {
+    return 0;
+  } else {
+    const maxLoops = input;
```

Comment:
Issue: It's generally not correct to assign a `const` to a variable value, instead you should use a `let` here

Reply:
Thanks Michael, is there not a school of thought that we should use const wherever we can so that the value is not accidentally overwritten by someone else? For the run of this function and with the declaration scope being inside the function, I thought maxLoops should not be allowed to change? I'll bring this up at our next buddy meeting to get your thoughts as I'm a bit unclear on this.